File: Python/1131 - Grenais.py
answer=1
grenais=0
v_inter=0
v_gremio=0
empate=0
while answer == 1:
    g_inter,g_gre=map(int,input().split())
    print("Novo grenal (1-sim 2-nao)")
    answer=int(input())
    grenais+=answer
    if g_inter>g_gre:
        v_inter+=1
    if g_gre>g_inter:
        v_gremio+=1
    if g_inter==g_gre:
        empate+=1        
else:
    print(f"{grenais-1} grenais")
    print(f"Inter:{v_inter}")
    print(f"Gremio:{v_gremio}")
    print(f"Empates:{empate}")
    if v_inter>v_gremio:
        print("Inter venceu mais")
    else:
        print("Gremio venceu mais")

File: Java/1131 - Grenais.java
import java.util.Scanner;
public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int golInter = 0,golGre = 0,vInt = 0, vGre = 0, empate = 0, resposta = 1,grenal = 0;
		while (resposta == 1) {
		    golInter = sc.nextInt();
		    golGre = sc.nextInt();
		    if (golInter == golGre)
		        empate++;
		    else if (golInter > golGre)
		        vInt++;
		    else    
		        vGre++;
		    grenal++;
		    System.out.println("Novo grenal (1-sim 2-nao)");
		    resposta = sc.nextInt();
		}
		String maiorVencedor;
		if (vInt > vGre)
		    maiorVencedor = "Inter";
		else
		    maiorVencedor = "Gremio";
		System.out.printf("%d grenais\nInter:%d\nGremio:%d\nEmpates:%d\n%s venceu mais\n",grenal,vInt,vGre,empate,maiorVencedor);
	}
}
